openstates/oh/bills.py

```diff
@@ -13,43 +13,333 @@
 class OHBillScraper(BillScraper):
     jurisdiction = 'oh'
 
-    def scrape(self, chamber, session):
+    def scrape(self, session, chambers):
         if int(session) < 128:
             raise AssertionError("No data for period {}".format(session))
 
-        base_url = 'http://www.lsc.state.oh.us/status%s/' % session
+        elif int(session) < 131:
+            #they changed their data format starting in 131st and added
+            #an undocumented API
+            self.old_scrape(session)
 
-        bill_types = {'lower': [('hb', 'bill'),
-                                ('hjr', 'joint resolution'),
-                                ('hcr', 'concurrent resolution')],
-                      'upper': [('sb', 'bill'),
-                                ('sjr', 'joint resolution'),
-                                ('scr', 'concurrent resolution')]}
 
-        for bill_prefix, bill_type in bill_types[chamber]:
-            url = base_url + '%s.xlsx' % bill_prefix
+            
+        else:
+            chamber_dict = {"Senate":"upper","House":"lower","House of Representatives":"lower","house":"lower","senate":"upper"}
+            
+            #so presumanbly not everything passes, but we haven't
+            #seen anything not pass yet, so we'll need to wait
+            #till it fails and get the right language in here
+            vote_results = {"approved":True,
+                            "passed":True,
+                            "adopted":True,
+                            "true":True,
+                            "false":False,
+                            True:True,
+                            False:False}
+
+            action_dict = {"ref_ctte_100":"committee:referred",
+                            "intro_100":"bill:introduced",
+                            "pass_300":"bill:passed",
+                            "intro_110":"bill:reading:1",
+                            "refer_210":"committee:referred",
+                            "crpt_301":"other",
+                            "crpt_317": "other",
+                            "intro_102":["bill:introduced","bill:passed"],
+                            "intro_105":["bill:introduced","bill:passed"],
+                            "intro_ref_ctte_100":"committee:referred",
+                            "refer_209":"other",
+                            "intro_108":["bill:introduced","bill:passed"],
+                            "intro_103": ["bill:introduced","bill:passed"],
+                            "msg_reso_503": "bill:passed",
+                            "intro_107": ["bill:introduced","bill:passed"],
+                            "imm_consid_360": "bill:passed"}
+
+            
+
+            base_url = "http://search-prod.lis.state.oh.us/"
```

Comment:
Some of the URLs created have double-slashes: one slash from the end of `base_url` and another from the beginning of the path they're accessing. Maybe delete one or the other?